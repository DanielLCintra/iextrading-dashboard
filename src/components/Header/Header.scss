.Header {
  padding: 30px 0;
  text-align: center;

  &Link {
    padding: 15px;
    margin-bottom: 15px;
    display: inline-block;
  }

  &Logo {
    width: 50px;
  }
}
