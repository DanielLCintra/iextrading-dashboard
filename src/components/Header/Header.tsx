import Typography from '@material-ui/core/Typography';
import * as React from 'react';
import classes from './Header.scss';

class Header extends React.Component {
  public render() {
    return (
      <header className={classes.Header}>
        <Typography
          className={classes.HeaderTitle}
          variant="display3"
          component="h1"
        >
          IEX Trading
          <Typography
            className={classes.HeaderSubTitle}
            variant="display1"
            component="small"
          >
            Bigest stocks
          </Typography>
        </Typography>
      </header>
    );
  }
}

export default Header;
