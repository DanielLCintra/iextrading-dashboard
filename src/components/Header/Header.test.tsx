import { shallow } from 'enzyme';
import * as React from 'react';
import Header from './Header';

describe('Header', () => {
  const wrapper = shallow(<Header />);

  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
