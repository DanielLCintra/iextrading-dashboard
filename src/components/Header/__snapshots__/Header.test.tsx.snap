// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without crashing 1`] = `
<header>
  <Link
    replace={false}
    title="Go to home"
    to="/"
  >
    <img
      alt="IEX Trading"
    />
  </Link>
  <WithStyles(Typography)
    component="h1"
    variant="display3"
  >
    IEX Trading
    <WithStyles(Typography)
      component="small"
      variant="display1"
    >
      Bigest stocks
    </WithStyles(Typography)>
  </WithStyles(Typography)>
  <Connect(SymbolSearch) />
</header>
`;
